meeting_scheduler/meetingscheduler/src/main/java/com/spring/rest/meetingscheduler/controller/MeetingController.java

```diff
@@ -35,6 +41,7 @@ public HashMap<String,Integer> getAvailability(@RequestBody MeetingDetail meetin
         return map;
     }
 
+//    Create meeting
     @PostMapping("/meeting")
```

Comment:
Request mappings can be used for shared mappings incase we have same route names and different HTTP methods. For simpler controllers it is better to use specific HTTP annotations as it provides more readability. So I have used @GetMapping, @PostMapping instead of @RequestMappings